Examples/Sources/MacroExamples/Interface/AccessorMacros.swift

```diff
@@ -0,0 +1,20 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftUI
```

Comment:
The example won’t compile on Linux and Windows if it depends on `SwiftUI`.

We’ve got two options here: Either wrap the macro in `#if canImport(SwiftUI)` or change it so it doesn’t depend on `SwiftUI`. Which one would you prefer?

Reply:
Thank you for proposing two options!
I'll choose the `#if canImport(SwiftUI)` option this time 🙏 

https://github.com/apple/swift-syntax/pull/2565/commits/b795cd867948fdefedd7c35958f84ab4a762e767